requests/models.py

```diff
@@ -685,6 +685,8 @@ def generate():
 
         if self._content_consumed and isinstance(self._content, bool):
             raise StreamConsumedError()
+        elif not isinstance(chunk_size, int):
+            raise TypeError("Value of chunk_size must be of type int.")
```

Comment:
Ok, things should be reworded. I didn't line wrap, as per the contributing docs, but let me know if it needs any other formatting tweaks.